src/Resources/views/storefront/page/search/layout/leftFilterLayout.html.twig

```diff
@@ -10,17 +10,18 @@ is configured on the left side.
     {% set productColumns = 'col-lg-8 col-xl-9' %}
 {% endif %}
 
+{# This value is explicitely checked for versions <6.4.4.0, instead of using the sidebar value #}
+{% if block is not defined %}
+    {% set block = { sectionPosition: 'sidebar' } %}
+{% else %}
+    {% set block = block|merge({ sectionPosition: 'sidebar' }) %}
+{% endif %}
+
 <div class="row">
-    <div class="cms-element-sidebar-filter {{ filterColumns }}">
-        <div id="filter-panel-wrapper"
-             class="filter-panel-wrapper"
-             data-offcanvas-filter-content="true">
-            {% sw_include '@Storefront/storefront/component/listing/filter-panel.html.twig' with {
-                listing: page.listing,
-                sidebar: true
-            } %}
-        </div>
-    </div>
+    {% sw_include '@Storefront/storefront/element/cms-element-sidebar-filter.html.twig' with {
```

Comment:
Does this show the mobile filter also for < 6.4.4?

Reply:
Yes

Reply:
Awesome!